bin/apisix

```diff
@@ -410,6 +410,7 @@ http {
             set $upstream_hdr_cache_control     '';
 
             proxy_cache                         $upstream_cache_zone;
+            proxy_cache_valid                   502 504 0s;
```

Comment:
We already have the "cache_http_status" property, so I think we can delete this line.

Reply:
Yes. here will be confusing. If 502 returned from upstream, the proxy_no_cache directive will take effect. But it returned from APISIX,  the proxy_no_cache directive will not take effect.

Reply:
I will remove it and explain why in the document.